contribs/gnofaucet/main_test.go

```diff
@@ -1 +1,156 @@
 package main
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/alicebob/miniredis/v2"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const defaultAccount_Seed = "source bonus chronic canvas draft south burst lottery vacant surface solve popular case indicate oppose farm nothing bullet exhibit title speed wink action roast"
+
+func TestServe(t *testing.T) {
+	t.Run("Serve without subcommand", func(t *testing.T) {
+		cmd := newServeCmd()
+		args := []string{}
+
+		// Run the command
+		cmdErr := cmd.ParseAndRun(context.Background(), args)
+		assert.ErrorContains(t, cmdErr, "help requested")
+	})
+
+	t.Run("Serve captcha without captcha-secret", func(t *testing.T) {
```

Comment:
These should go in `captcha_test.go`

Reply:
Thanks, separated on captcha_test & github_test
cb07372f4406162a7eacfd910e7551c83f540616